hazelcast/src/test/java/com/hazelcast/replicatedmap/ReplicatedMapMergePolicyTest.java

```diff
@@ -57,11 +54,6 @@
 @Category(NightlyTest.class)
```

Comment:
I couldn't be sure about it because it still takes more than a minute to complete all the tests but I can do it if it is ok